### ISSUE TITLE
I cannot find the options to activate

### ISSUE THREAD

**Issue by martatorredev** [WordPress/wporg-two-factor#222]
I read that it was this link https://wordpress.org/support/users/username/edit/account

But I can't find anything to change it. I can edit the forum profile and the .org profile but I don't see anything about the two factor.

**Comment by iandunn** [comment]
🤔 I got a fatal JS error on the page when I loaded it just now, but then it worked after I refreshed.

Do you mind trying that?

If that fixes it, we should look into why the `?ver=...` cachebusters aren't working.

```
react-dom.min.js?ver=18:1 TypeError: Cannot read properties of null (reading 'replace')
    at i (script.js?ver=95382bb840bece602d5f:1:1540)
    at wt (react-dom.min.js?ver=18:1:47623)
    at $s (react-dom.min.js?ver=18:1:120736)
    at wl (react-dom.min.js?ver=18:1:88644)
    at bl (react-dom.min.js?ver=18:1:88572)
    at yl (react-dom.min.js?ver=18:1:88435)
    at il (react-dom.min.js?ver=18:1:85259)
    at fl (react-dom.min.js?ver=18:1:85646)
    at zn (react-dom.min.js?ver=18:1:32460)
    at react-dom.min.js?ver=18:1:83211
ar @ react-dom.min.js?ver=18:1
react-dom.min.js?ver=18:1 Uncaught TypeError: Cannot read properties of null (reading 'replace')
    at i (script.js?ver=95382bb840bece602d5f:1:1540)
    at wt (react-dom.min.js?ver=18:1:47623)
    at $s (react-dom.min.js?ver=18:1:120736)
    at wl (react-dom.min.js?ver=18:1:88644)
    at bl (react-dom.min.js?ver=18:1:88572)
    at yl (react-dom.min.js?ver=18:1:88435)
    at il (react-dom.min.js?ver=18:1:85259)
    at fl (react-dom.min.js?ver=18:1:85646)
    at zn (react-dom.min.js?ver=18:1:32460)
    at react-dom.min.js?ver=18:1:83211
```

**Comment by StevenDufresne** [comment]
I've ran into this periodically as well and seems like a logic error. For some reason `screen` is passed in as null. We grab the  screen from the search params:

https://github.com/WordPress/wporg-two-factor/blob/881cb34f82d5689dd20e3b18774a60cbf5498108/settings/src/script.js#L66

Use a ternary to determine if we have a screen, if we don't we use the `ScreenNavigation` component, pass in screen, but them run a replace on it.

https://github.com/WordPress/wporg-two-factor/blob/881cb34f82d5689dd20e3b18774a60cbf5498108/settings/src/script.js#L145

It probably works on reload because we've adjusted the URL in the meantime. I may have that wrong though. That's a quick look at the code without testing anything.
